images/public-log-vendor-matcher/pg-init.d/lib-07-bq-load-logs.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Load the k8s-infra-artifacts-gcslogs logs into the usage_all_raw dataset
+if [ -z "${GCP_BIGQUERY_DATASET_LOGS:-}" ]; then
+    echo "Using dataset logs, since \$GCP_BIGQUERY_DATASET_LOGS was provided and set to '${GCP_BIGQUERY_DATASET_LOGS:-}'"
+    BUCKETS=(
```

Comment:
still missing staging buckets here but could be done as followup

Reply:
There's now a script to manually generate the list of buckets to pull usage from (as `${BUCKET}_usage`)

https://github.com/kubernetes/k8s.io/pull/2710/commits/98bd4dba50c0613eb7304e20fb6413cc93e7efc2